lib/ancestry/class_methods.rb

```diff
@@ -27,6 +27,20 @@ def orphan_strategy= orphan_strategy
       end
     end
 
+    # Delimiter writer
+    def ancestry_delimiter= ancestry_delimiter
+      # Check value of the delimiter. Restrict if it's numeric.
+      if ancestry_delimiter !~ /^#{ancestry_id_pattern}$/
```

Comment:
This should probably be `/\A#{ancestry_id_pattern}\z/` so as not to match around newlines.